UnityProject/Assets/Scripts/ScriptableObjects/TimedGameEvents/HalloweenGhosts.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Systems.Spawns;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace ScriptableObjects.TimedGameEvents
+{
+	[CreateAssetMenu(fileName = "HalloweenGhostsTimedEvent", menuName = "ScriptableObjects/Events/TimedGameEvents/HalloweenGhosts")]
+	public class HalloweenGhosts : TimedGameEventSO
+	{
+		[SerializeField] private List<GameObject> horrorsToSpawn;
+		[SerializeField] private SpawnPointCategory spawnPointCategory = SpawnPointCategory.MaintSpawns;
+		[SerializeField] private Vector2 randomSpawnCount = new Vector2(1, 5);
```

Comment:
`Vector2` is used for positions. The UI will show an X and Y label, which definitely isn't clear.

Can we use a `randomSpawnMin` and `randomSpawnMax` instead?

Reply:
We've been using vector2 for random values forever now
it takes less space in the inspector, and no one has ever complained about it being "confusing" in all the time I've been here.

Reply:
Strange but okay. How about `Vector2Int`?

Reply:
done